account_invoice_split_invoice/wizards/account_invoice_split_wizard.py

```diff
@@ -42,7 +41,7 @@ def split_invoice(self):
                 if was_open:
                     old_invoice.action_invoice_cancel()
                     old_invoice.action_invoice_draft()
-                    old_invoice.env.invalidate_all()
+                    self.env.clear()
```

Comment:
Why did you make this change?

Reply:
Because it is asked in the migration checklist:
https://confluence.compassion.ch/pages/viewpage.action?pageId=31719444

Reply:
Good point! ;-)

Reply:
I just saw that it should be old_invoice.env.clear... i'll correct that